README.md

```diff
@@ -1,4 +1,49 @@
-# README
+#What is Slow Food Online?
 
-## slow_food_online_june
-Craft Academy Bootcamp Week 8-9 Challenge
+Slow Food Online is a collaboration between 5 great minds, which attempts to fill the apparent void of user-friendly,
+interactive and well designed food ordering services.
+
+##Tools and technologies
+
+For this project we've used a bunch of quality tools and technologies which have aided us in the creation
+and delivery of this project.
+
+**Language**:
+* [Ruby](https://www.ruby-lang.org/)
+
+**Frameworks**:
+* [Ruby on Rails](http://rubyonrails.org/)
+
+**Planning**:
+* [Pivotal Tracker](https://www.pivotaltracker.com/n/projects/1609537)
+
+**Template engine**:
+* [Haml](http://haml.info/)
+
+**Database**:
+* [PostgreSQL](https://www.postgresql.org/)
+
+**Testing**:
+* [RSpec](http://rspec.info/)
+* [Cucumber](https://cucumber.io/)
+* [Coveralls](https://coveralls.io/)
+* [Travis CI](https://travis-ci.org/)
+
+## Installing
+Fork and Clone this repo.
+
+Once the source is on your machine run the following commands to install all the necessary packages and gems:
+
+```
+$ bundle install
+
+$ rake db:create
```

Comment:
maybe go with `rails db:create` -- not really important since `rake` still works with Rails 5